### PR TITLE
doc: add cephfs vsg to readme

### PR DESCRIPTION
**Pull request by Madhu-1**
added cephfs vsg feature to the readme https://docs.ceph.com/en/latest/cephfs/fs-volumes/#subvolume-quiesce

### CODE REVIEW

**Review by nixpanic — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ for its support details.
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Pacific (>=v16.2.0)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWOP volume | Alpha          | >= v3.5.0          | >= v1.5.0        | Pacific (>=v16.2.0)  | >= v1.22.0         |
 |        | Creating and deleting snapshot                            | GA             | >= v3.1.0          | >= v1.0.0        | Pacific (>=v16.2.0)  | >= v1.17.0         |
+|        | Creating and deleting volume group snapshot               | Alpha          | >= v3.11.0         | >= v1.9.0       | Squid   (>=v19.0.0)  | >= v1.31.0         |
```
Comment (Rakshith-R) [inline README.md:104]: We don't have support for RBD volume group snapshot yet ? 
Should we only mention CephFS here ?

Comment (Madhu-1) [reply README.md:104]: This is already in cephfs section, we have different section for rbd

Comment (Rakshith-R) [reply README.md:104]: Sorry, Missed it

**Review by Rakshith-R — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by nixpanic** [comment]
@mergifyio rebase

**Pull request merged**